sync/class.jetpack-sync-actions.php

```diff
@@ -340,3 +355,5 @@ static function init_sync_cron_jobs() {
 
 // We need to define this here so that it's hooked before `updating_jetpack_version` is called
 add_action( 'updating_jetpack_version', array( 'Jetpack_Sync_Actions', 'do_initial_sync' ), 10, 2 );
+
+add_action( 'plugins_loaded', array( 'Jetpack_Sync_Actions', 'setup_woocommerce' ) );
```

Comment:
is `plugins_loaded` isn't "early" enough?

performance wise maybe it is better just to initialise the module/listeners even if woo isn't installed?